frontend/packages/ceph-storage-plugin/integration-tests-cypress/views/store.ts

```diff
@@ -7,16 +7,23 @@ export enum Providers {
 
 export const testName = 'test-bucket';
 
+export enum StoreType {
+  BackingStore = 'backingstore',
+  NamespaceStore = 'namespacestore',
+}
+
+let storeType = StoreType.BackingStore;
```

Comment:
The functions (inputCustomSecrets, setupAWS, etc. )can individually take `storeType` as a parameter.
Or create a class instead where you can initialize this in the constructor.

Reply:
added parameters to the functions